pyanaconda/modules/storage/storage.py

```diff
@@ -177,6 +182,13 @@ def set_storage(self, storage):
         self.storage_changed.emit(storage)
         log.debug("The storage model has changed.")
 
+    def on_protected_devices_changed(self, protected_devices):
+        """Update the protected devices in the storage model."""
+        if not self._storage:
+            return
```

Comment:
Please at least log that nothing happened.

Reply:
I have added logging to the method `protect_devices` to log that something happened.